[Pull-based Ingestion] Update pull-based ingestion metrics and reset settings (#9993)
opensearch-project/documentation-website: _api-reference/document-apis/pull-based-ingestion-management.md

| `cluster_manager_timeout` | Time units | The amount of time to wait for a connection to the cluster manager node. Default is `30s`. |
| `timeout` | Time units | The amount of time to wait for a response from the cluster. Default is `30s`. |

### Request body

> A list of reset settings can be provided for all or a subset of shards that need to be reset. This is optional.

Reviewer: ```suggestion
The following table lists the available request body fields.
```

| Field | Data type | Required/Optional | Description |
| :--- | :--- | :--- | :--- |
| `shard` | Number | Required | The shard to reset. |

> | `mode` | String | Required | Reset mode. Allowed values are `offset` and `timestamp`. |

Reviewer: Please add a short description of each value in parentheses. Should we link to [this doc](https://docs.opensearch.org/docs/latest/api-reference/document-apis/pull-based-ingestion/#stream-position) for more information?
```suggestion
| `reset_settings.mode` | String | Required | The reset mode. Valid values are `offset` and `timestamp`. |
```

> | `value` | String | Required | &ensp;&#x2022; `offset`: Kafka offset or Kinesis sequence number<br>&ensp;&#x2022; `timestamp`: Unix timestamp in milliseconds |

Reviewer: ```suggestion
| `reset_settings.value` | String | Required | &ensp;&#x2022; `offset`: The Kafka offset or Kinesis sequence number<br>&ensp;&#x2022; `timestamp`: A Unix timestamp in milliseconds |
```

> ```json

Reviewer: This is not needed in this section because we have an example request.

Author: Good point. Removed it.
